env/git.go

```diff
@@ -112,26 +112,21 @@ func GitSHA(path string) (string, error) {
 }
 
 var GitBlob = func(path string, commit *object.Commit) (string, error) {
-	if commit == nil {
-		blob, err := fallbackBlob(path)
+	if commit != nil {
+		if file, err := commit.File(path); err == nil {
```

Comment:
right, my bad... i did say i was going to change that because it's super confusing. I just did.